tests/conftest.py

```diff
@@ -109,25 +105,21 @@ def collect_catalogs():
     target a specific cat_name
     """
 
-    def _get_origin_time(cat):
-        ori = obsplus.get_preferred(cat, "origin")
-        return ori.time
-
     out = {}
     for cat_path in catalogs:
         cat = obspy.read_events(cat_path)
         # sort events by origin time
-        cat.events.sort(key=_get_origin_time)
+        cat.events.sort(key=obsplus.get_reference_time)
         out[os.path.basename(cat_path).split(".")[0]] = cat
         # add to cache and assert cat_name resource id is unique
         cat_id = cat.resource_id.id
         assert cat_id not in eve_id_cache  # test catalogs must have unique id
         eve_id_cache[cat_id] = len(cat)
 
     # get catalog from datasets
-    out["kemmerer"] = obsplus.load_dataset("kemmerer").event_client.get_events()
-    out["bingham"] = obsplus.load_dataset("bingham").event_client.get_events()
-    out["crandall"] = obsplus.load_dataset("crandall").event_client.get_events()
+    for name in ("bingham_test", "crandall_test"):
+        ds = obsplus.load_dataset(name)
```

Comment:
It looks like you've put a lot of work into this!

I tried to download the branch and run the tests and the tests are refusing to run, with the following error:
`ImportError while loading conftest '../obsplus/tests/conftest.py'.
tests/conftest.py:155: in <module>
    cat_dict = collect_catalogs()
tests/conftest.py:121: in collect_catalogs
    ds = obsplus.load_dataset(name)
obsplus/datasets/dataset.py:305: in load_dataset
    raise ValueError(msg)
E   ValueError: bingham_test is not in the known datasets []`

I've tried to step through this, but there's some magic going on with entry points that I can't quite figure out.

Reply:
Update: deleting and re-installing obsplus appears to have resolved this. But it might be good to figure out if there is a way to avoid this?

Reply:
Unfortunately no, the entry points only get registered when the setup.py files runs.